src/scss/base/_global.scss

```
@@ -0,0 +1,23 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  overflow-x: hidden;
+  font-family: $ff-time;
+  font-size: $fs-sm;
+  color: $color-white;
+}
+ul {
```

```suggestion
}

body {
  overflow-x: hidden;
  font-family: $ff-time;
  font-size: $fs-sm;
  color: $color-white;
}

ul {
```
Mỗi block cách nhau 1 dòng nhé